tubular/base.py

```diff
@@ -53,6 +53,7 @@ def classname(self):
         return type(self).__name__
 
     def __init__(self, columns=None, copy=True, verbose=False) -> None:
+        """Initializes instance with columns and whether to make a copy or not from inputs."""
```

Comment:
This and other init docstrings should specify what they initialise. i.e. "Initialize a BaseTransformer with columns to transform and whether to make a copy"